src/OrchardCore.Modules/OrchardCore.Forms/Workflows/Activities/BindModelStateTask.cs

```diff
@@ -42,7 +42,22 @@ public override ActivityExecutionResult Execute(WorkflowExecutionContext workflo
 
         foreach (var item in httpContext.Request.Form)
         {
-            updater.ModelState.SetModelValue(item.Key, item.Value, item.Value);
+            // Avoid creating a new array for rawValue if there's only one value.
+            object rawValue;
+            if (item.Value.Count == 0)
```

Comment:
Can we use switch expression instead?

Reply:
That's just harder to read and write for me 🙈 And I don't see any benefit...